src/huggingface_hub/hf_api.py

```diff
@@ -2714,6 +2722,19 @@ def upload_folder(
         ... )
         # "https://huggingface.co/datasets/username/my-dataset/tree/main/remote/experiment/checkpoints"
 
+        # Upload checkpoints folder including logs while deleting existing logs from the repo
+        # Useful if you don't know exactly which log files have already being pushed
+        >>> upload_folder(
+        ...     folder_path="local/checkpoints",
+        ...     path_in_repo="remote/experiment/checkpoints",
+        ...     repo_id="username/my-dataset",
+        ...     repo_type="datasets",
+        ...     token="my_token",
+        ...     delete_patterns="**/logs/*.txt",
+        ... )
+        # "https://huggingface.co/datasets/username/my-dataset/tree/main/remote/experiment/checkpoints"
```

Comment:
If this is the return of the method above, I don't think you need to precede it with `#`. I thought this was a comment for clarity but I believe it's actually the return of the method?

Reply:
```suggestion
        "https://huggingface.co/datasets/username/my-dataset/tree/main/remote/experiment/checkpoints"
```

True, thanks for noticing